src/main/groovy/com/bmuschko/gradle/clover/CloverPlugin.groovy

```diff
@@ -122,29 +131,23 @@ class CloverPlugin implements Plugin<Project> {
             test.include optimizeTestSetAction // action is also a file inclusion spec
             test.doLast createCreateSnapshotAction(cloverPluginConvention, project, test)
             if (project.hasProperty('cloverInstrumentedJar')) {
+
                 log.info "Skipping RestoreOriginalClassesAction for {} to generate instrumented JAR", test
```

Comment:
This log statement is no longer true. Delete it?

Reply:
Yes, this was eventually removed by df90bd7f9abbd8c7b202ed108755fccf6900cafc